getting_started/setup_vm/roles/ccf_install/tasks/deb_install.yml

```diff
@@ -1,9 +1,19 @@
 - name: Include vars
   include_vars: common.yml
 
+- name: Get package url
```

Comment:
I think you could have left the var for versioned ccf_ver case, and just added the curl gated with a `when: "{{ ccf_ver }} = "latest"`, trading off an extra line of YAML in return for keeping the bash a one-liner.

The end result is exactly the same of course.

Reply:
Yeah I thought about that but I wanted the URL construction for both cases in a single place. Also you have the issue that in one case you need `.stdout`.

Reply:
Ah true, you'd have needed a separate `set_fact`, with the same `when` to assign the `.stdout` to the same name.